packages/subgraph/schemas/schema.v1.graphql

```diff
@@ -84,7 +84,7 @@ type Poll @entity {
   maxMessages: BigInt!
   maxVoteOption: BigInt!
   messageProcessor: Bytes! # address
-  tally: Bytes! # address
+  tally: Tally! # address
```

Comment:
it's not an address anymore?

Reply:
It's Tally entity now. Right now, there will be only tally results (there are no total spent, voice credits per option, commitments and etc).

Reply:
Yes, so I think the `#address` should be removed?